.vscode/settings.json

```diff
@@ -4,5 +4,5 @@
 		"source.fixAll.eslint": true
 	},
 	"files.eol": "\n",
-	"cSpell.words": ["Mitchells"]
+	"cSpell.words": ["inhouse", "Mitchells"]
```

Comment:
It may not be ideal to have vscode settings here, maybe remove .vscode from git history